input/emote-acquisition/story.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "story",
+  "contents": [
+    // End of Dragons
+    "Paper","Rock","Scissors"
```

Comment:
Given these aren't in the API, how sure are you that their ids wouldn't be "paper","rock","scissors" instead (code is case sensitive)?

Reply:
Yeah, those are copied from the group file, but either the API got changed or they were temporary references.

Do as you wish, keep or delete. I'd keep them around if they don't give any problems, then change them later once ArenaNet updates the API.

Reply:
Ah, ok. I wasn't sure where they had come from. Odd they disappeared from the api. No harm in keeping them.

Reply:
I might have taken them from the game itself too, don't remember. The emotes api is outdated too anyway, so we'll fix them down the road, there aren't many of them after all.

Personally, I'm crossing my fingers for ArenaNet to move them into the Novelties API and use numeric ids for them as well lol.